File: sdk/csharp/src/Hjtpx.Captcha.Sdk/Exceptions/AuthenticationException.cs
namespace Hjtpx.Captcha.Exceptions;

public class AuthenticationException : CaptchaException
{
    public AuthenticationException(string message)
        : base(message, "AUTH_ERROR", false)
    {
    }

    public AuthenticationException(string message, Exception innerException)
        : base(message, "AUTH_ERROR", false, innerException)
    {
    }
}

File: sdk/java/src/main/java/com/hjtpx/captcha/exception/AuthenticationException.java
package com.hjtpx.captcha.exception;

public class AuthenticationException extends CaptchaException {
    public AuthenticationException(String message) {
        super(message, "AUTHENTICATION_ERROR", false);
    }

    public AuthenticationException(String message, Throwable cause) {
        super(message, "AUTHENTICATION_ERROR", false, cause);
    }
}
